demo/timing.c

```diff
@@ -126,15 +126,15 @@ int main(void)
       SLEEP;
       mp_rand(&a, cnt);
       mp_rand(&b, cnt);
-      rr = 0;
-      tt = -1;
+      rr = 0u;
+      tt = UINT64_MAX ;
```

Comment:
\*plenk\* :)

doesn't your linter complain about that space before the `;`?

Reply:
@sjaeckel good catch